screens/FollowerProfile.tsx

```diff
@@ -0,0 +1,170 @@
+  { name: "kevin", source: require("../assets/images/Kevin.jpeg") },
+  { name: "luca", source: require("../assets/images/Luca.jpeg") },
+  { name: "max", source: require("../assets/images/Max.jpeg") },
+];
+
+type PostProps = {
+  image: string;
+};
+
+const Post: React.FC<PostProps> = ({ image }) => (
+  <Image source={{ uri: image }} style={styles.post} />
+);
+
+type User = {
+  id: string;
+  username: string;
+  nickname: string;
+  avatar: string;
+  posts: string[];
+  isFollowing: boolean;
+};
+
+interface RouteParams {
+  user?: User;
+}
+
+function isUser(obj: any): obj is User {
+  return (
+    obj &&
+    typeof obj.id === "string" &&
+    typeof obj.username === "string" &&
+    typeof obj.nickname === "string" &&
+    typeof obj.avatar === "string" &&
+    Array.isArray(obj.posts) &&
+    typeof obj.isFollowing === "boolean"
+  );
+}
+
+const FollowerProfile = () => {
+  const route = useRoute();
+
+  const params = route.params as RouteParams;
+  const user = params.user && isUser(params.user) ? params.user : null;
+
+  if (!user) {
+    return <Text>User nicht gefunden</Text>;
+  } else
+    return (
+      <ScrollView style={styles.container}>
+        <Image source={{ uri: user.avatar }} style={styles.avatar} />
+
+        <View style={styles.header}>
+          <View style={styles.userInfo}>
+            <Text style={styles.name}>{user.nickname}</Text>
+            <Text
+              style={{
+                fontSize: SIZES.medium,
+                color: COLORS.darkgray,
+                marginBottom: "3%",
+              }}
+            >
+              {user.username}
+            </Text>
+            <Text style={{ marginBottom: 12, marginTop: 6 }}>
+              Hip Hip Hurra, mein Lumberjack ist da!
+            </Text>
+          </View>
+          <View style={styles.buttonArea}>
+            <TouchableOpacity
+              style={styles.button}
+              onPress={() => console.log("entfolgen")}
+            >
+              <Text style={{ fontSize: SIZES.medium }}>entfolgen</Text>
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={styles.button}
+              onPress={() => console.log("chatten")}
+            >
+              <Text style={{ fontSize: SIZES.medium }}>chatten</Text>
+              <Ionicons name="chatbubbles" style={{ marginLeft: 10 }} />
+            </TouchableOpacity>
+          </View>
+        </View>
+        <Text
+          style={{
+            fontSize: SIZES.large,
+            fontWeight: "bold",
+            marginTop: "5%",
+            marginLeft: "4%",
+            marginBottom: "1%",
+          }}
+        >
+          Beiträge
+        </Text>
+        <View style={styles.posts}>
+          {user.posts.map((url: any, index: any) => (
+            <Image key={index} source={url} style={styles.post} />
+          ))}
+        </View>
+      </ScrollView>
+    );
+};
+export default FollowerProfile;
+
+const styles = StyleSheet.create({
```

Comment:
kann man das auch in ein externes Stylesheet packen? Dann würde das in "Stylesheets" ordner gehören

Reply:
ja, würde das aber generell einmal einheitlich festlegen und einordnen zusammen. Auf einem eigenen Branch und einheitlich für alle stylesheets in den tsx files. Bis dahin würde ich die styles erstmal in den jeweiligen files lassen, sonst wirds schnell unübersichtlich